docs/blink-codes.md

```diff
@@ -0,0 +1,53 @@
+# Fobnail blink codes
+
+One of the main advantages of using hardware dongle for attestation is that it
+can report the state to users without relying on proper functioning of PC (or
+another device) being attested. This is done by various blink codes representing
+different states of Fobnail Token operation.
+
+![nRF52840 dongle LEDs location](images/nRF52840_dongle_buttons_leds.png)
+
+Even though LED2 on nRF52840 dongle is a multicolor RGB LED, Fobnail only uses
+red in order to not exclude users with impaired color vision. LED1 is green.
+Unless noted otherwise, short blinks and intervals between blinks take 100 ms.
+
+## Idle states
+
+#### Token not provisioned
+
+Red LED will be lit indefinitely.
+
+#### Token provisioned, waiting for CoAP client
+
+Both LEDs will shortly blink every 5 seconds. Cycle starts with blink, followed
+by a waiting period, so any LED failure should be visible when provisioned Token
+is connected to the device.
+
+## Fobnail provisioning
+
+On successful Token provisioning, green LED will blink once. No change on
+failure, i.e. red light will be kept on.
+
+## Platform provisioning
+
+On successful Token provisioning, green LED will blink three times. After
+failure followed by client disconnection Token goes back to waiting state, i.e.
+both LEDs will blink every 5 seconds.
```

Comment:
```suggestion
On successful Token provisioning, green LED will blink three times, then token goes back to waiting state. On failure no signal will be given, the token will stay in waiting state and will keep blinking both LEDs every 5 seconds.
```
May require adding line break manually.

Reply:
Updated. I also changed `Token provisioning` -> `platform provisioning`, this was a copy-paste error.